destreamer.ts

```diff
@@ -247,9 +269,36 @@ async function getVideoInfo(videoID: string, session: any) {
             return playbackUrl;
         });
 
-    return [title, hlsUrl];
+    return [title, date, hlsUrl];
 }
 
+function getVideoUrls() {
+    if (argv.videoUrls.length === 0)

```

Comment:
maybe here we should trow an error if the length of the array is 0, we have no urls (even if i think that being a required arg it cannot be empty)

Reply:
You are right, removed :)